contracts/v2/PolygonZkEVMBridgeV2.sol

```diff
@@ -248,11 +251,16 @@ contract PolygonZkEVMBridgeV2 is
                 ];
 
                 if (tokenInfo.originTokenAddress != address(0)) {
-                    // The token is a wrapped token from another network
-
-                    // Burn tokens
-                    TokenWrapped(token).burn(msg.sender, amount);
-
+                    // The token is either (1) a wrapped token from another network
```

Comment:
these changes might be unnecessary (or maybe me and paul are missing some obvious which is very possible 😄 )

assume bridgeAsset(L2_USDCe, destination=L1, ...), then token=L2_USDCe

what is wrapper? with the suggested changes below, wrapper==L2_USDCe => wrapper==token

without the changes below, wrapper==wrappedTokenAddress

so what is wrappedTokenAddress? 
if wrappedTokenAddress==L2_BWUSDC, then that's wrong, because it was never minted in the first place 
if wrappedTokenAddress==L2_USDCe, then... wrappedTokenAddress==token